workflows/api/types/workflow_run.py

```diff
@@ -625,8 +626,6 @@ async def update_workflow_run(
         )
         if not deprecated_by:
             raise PlatformicsException("Unauthorized: deprecated_by does not exist")
-        params["deprecated_by"] = deprecated_by[0]
-        del params["deprecated_by_id"]
```

Comment:
I removed these lines because this was causing some unexpected behavior:
* I have two `workflowRuns`, one ending in `6397d` (status ABORTED) and one ending in `d80a8` (status CREATED)
* In my update mutation, I'm trying to update `6397d` to be deprecated by `d80a8`
* But after running the mutation, it looks like the reverse has happened, even though the `updatedAt` timestamp was set on the correct `workflowRun`

<img width="1234" alt="Screenshot 2024-04-18 at 4 03 11 PM" src="https://github.com/chanzuckerberg/czid-platformics/assets/53838890/52795f4b-39b5-4b04-8866-4256558a49af">
<img width="1382" alt="Screenshot 2024-04-18 at 4 03 16 PM" src="https://github.com/chanzuckerberg/czid-platformics/assets/53838890/86ecd743-5adb-4f0a-8c53-74680cbca30d">

Removing these lines allows us to update the correct `workflowRun`'s `deprecatedById` value as we expect (see the example in the PR description).

To be completely honest, I'm not sure why these lines were causing the incorrect `workflowRun` to be updated. I tried stepping through the code and it looked like all the correct objects were being fetched/updated with the right params, so I'm not sure why updating the `id` instead of the actual related object fixes it.

My best guess is that something funky is going on with the underlying logic because `deprecatedBy` causes `workflowRun` to have a 1:1 relationship with itself. Let me know if there are concerns with this change or if there are any other solutions to fix this!